src/server_creator.py

```diff
@@ -226,10 +226,11 @@ def __init__(self):
         cprint(f"&aServer created at {server_path=}")
         # Send command to hook into proxy
         if 'paper' in JAR_NAME.lower() and isBehindBungee == "true":
-            if input("Enable Firewall for this server? ([y]/n)").startswith('y'):
-                print("Is firewall enabled", Firewall().isFirewallEnabled())
-                Firewall().denyPort(port)
-                cprint(f"\n&cFirewall has been denied port to {port}")
+            if os.geteuid() == 0:           
+                if input("Enable Firewall for this server? ([y]/n)").startswith('y'):
```

Comment:
please use cinput (since I hadded error handling for ctrl+c)

Reply:
that was ur mistake clown lol I switch it tho